File: .sdk/tm/php/utility/MakeContext.php
<?php
declare(strict_types=1);

// FuelPricesAtSpanishGasStations SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class FuelPricesAtSpanishGasStationsMakeContext
{
    public static function call(array $ctxmap, ?FuelPricesAtSpanishGasStationsContext $basectx): FuelPricesAtSpanishGasStationsContext
    {
        return new FuelPricesAtSpanishGasStationsContext($ctxmap, $basectx);
    }
}

File: .sdk/tm/rb/utility/make_context.rb
# FuelPricesAtSpanishGasStations SDK utility: make_context
require_relative '../core/context'
module FuelPricesAtSpanishGasStationsUtilities
  MakeContext = ->(ctxmap, basectx) {
    FuelPricesAtSpanishGasStationsContext.new(ctxmap, basectx)
  }
end
